pkgdb/src/resolver/environment.cc

```diff
@@ -391,12 +391,9 @@ Environment::lockPackage( const LockedInputRaw & input,
   LockedPackageRaw pkg;
   pkg.input = input;
   info.at( "absPath" ).get_to( pkg.attrPath );
-  info.erase( "id" );
-  info.erase( "description" );
   info.erase( "absPath" );
-  info.erase( "subtree" );
+  info.erase( "id" );
   info.erase( "system" );
-  info.erase( "relPath" );
```

Comment:
question nonblocking: are these fields accessible in the lockfile if you use the manifest portion of the file? Do we want to duplicate them?

Also I'm guessing this is the reason for the `relPath` errors. I guess this would call for a lockfile versioning bump, but pointing that out more because we should do it if we make similar changes in the future, probably not right now.

Reply:
I'm infering them now on the rust side, keep the number of exposed fields lower.